crates/next-dev-tests/tests/integration/next/app/metadata/input/next.config.js

```diff
@@ -2,4 +2,5 @@ module.exports = {
   experimental: {
     appDir: true,
   },
+  transpilePackages: ["@turbo/pack-test-harness"],
```

Comment:
Any idea why I need to include this for this test, but it's not necessary for the 404 test?

Reply:
🤷

Reply:
This was because these two cases have a `tsconfig.json` that overrides the top-level `tsconfig.json`, which includes the test harness in its paths list (hence it's compiled without needing transpilePackages).